src/wrappers/jit.rs

```diff
@@ -9,6 +9,7 @@ use torch_sys::*;
 
 /// Argument and output values for JIT models.
 #[derive(Debug, PartialEq)]
+#[non_exhaustive]
```

Comment:
https://doc.rust-lang.org/reference/attributes/type_system.html#the-non_exhaustive-attribute